keras_rs/src/layers/retrieval/sampling_probability_correction.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+import keras
+from keras import ops
+
+from keras_rs.src import types
+from keras_rs.src.api_export import keras_rs_export
+
+
+@keras_rs_export("keras_rs.layers.SamplingProbablityCorrection")
+class SamplingProbablityCorrection(keras.layers.Layer):
+    """Sampling probability correction.
+
+    Args:
+        epsilon: Small float added to sampling probability to avoid taking the
```

Comment:
I've been thinking about our doc-strings. In the rest of Keras, the format is `{arg_name}: {dtype}. {description}`. For example, https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/f_net/f_net_backbone.py#L41.
I wonder if we should do the same. Or because we are doing static typing, it'll show up in the generated documentation anyway?

Reply:
Oh, that's a good question. I don't know if the keras.io scripts handle type annotations.

In code completion, you see the arguments with their types and default values. That's good. Except that `types.Tensor` for instance is replaced by `Any`, because that's how it's defined to not depend on all backends. So maybe for that reason too it makes sense to add the types here.

Reply:
Added.